Gemfile

```diff
@@ -0,0 +1,51 @@
+source 'https://rubygems.org'
+git_source(:github) { |repo| "https://github.com/#{repo}.git" }
+
+ruby '2.7.0'
```

Comment:
use `"` instead of `'`
remove all comment in this file

Reply:
done

Reply:
I don't see your changes.